ReactiveFeedback/FeedbackEventCustomer.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public class FeedbackEventConsumer<Event> {
+    struct Token: Equatable {
+        let value: UUID
+
+        init() {
+            value = UUID()
+        }
+    }
+
+    func process(_ event: Event, for token: Token) {
+        fatalError()
```

Comment:
[nit] would be nice to provide a message here
```suggestion
        fatalError("This is an abstract class. You must subclass this and provide your own implementation")
```